### PR TITLE
21 create adminmovie agent dashboard co

### PR DESCRIPTION
**Pull request by cylia93**
dashbord

### CODE REVIEW

File: .vscode/settings.json
Comment (Emimint) [inline .vscode/settings.json:1]: Please add to `.gitignore` file

File: admin-agent-movie.html
Comment (Emimint) [inline admin-agent-movie.html:1]: Mobile version not working : 

image

**Review by Emimint — CHANGES_REQUESTED** [review]
@cylia93 la version mobile marche pas et le `settings.json` ne devrait pas faire parti des commits. 

image

**Comment by Emimint** [comment]
@cylia93 @Doupmat  till need to be fixed later, but we can do that when we do the final layout : 


iPhone :

image

sidebar, iPhone : 

image

**Review by Emimint — APPROVED** [review]

**Review by Doupmat — APPROVED** [review]
bon

**Pull request merged**